script/dev-deploy

```diff
@@ -1265,6 +1418,12 @@ class SimpleService(Service):
             "store_admin_host": self.admin_hosts[0][1],
             "config_path": self.hserver_remote_conf,
             "restart": False if "hserver" in disable_restarted else True,
+            "enable_tls": self.hserver_enable_tls,
+            "openssl_cnf": self.openssl_cnf,
+            "ca_key_path": self.ca_key_path,
+            "ca_cert_path": self.ca_cert_path,
+            "ca_out_dir": self.ca_out_dir,
+            "ca_remote_dir": self.ca_remote_dir,
```

Comment:
There are so many arguments related to tls. Maybe you can use a map to collect all of them, also in python, [namedtuple](https://docs.python.org/3.8/library/collections.html?highlight=namedtuple#collections.namedtuple) is a good choice